src/client/sandbox/node/element.js

```diff
@@ -574,11 +576,18 @@ export default class ElementSandbox extends SandboxBase {
         el.setAttribute('target', storedAttr || el.target);
     }
 
-    _setValidBrowsingContextOnClick (el) {
-        el.addEventListener('click', () => {
+    _setValidBrowsingContextOnElementClick (window) {
+        window.addEventListener('click', e => {
```

Comment:
I suppose we should call `sandbox.addInternalEventListener` here instead `element.addEventListener` because now [this handler](https://github.com/DevExpress/testcafe-hammerhead/pull/897/files#diff-d02008885b9c21dddc064a025f83c5ceR79) will be called before yours